net/net.sh

```diff
@@ -294,7 +294,7 @@ startNode() {
       "
   ) >> "$logFile" 2>&1 &
   declare pid=$!
-  ln -sf "fullnode-$ipAddress.log" "$netLogDir/fullnode-$pid.log"
+  ln -sfT "fullnode-$ipAddress.log" "$netLogDir/fullnode-$pid.log"
```

Comment:
This was the original form, including the -T option.  I had commented it out previously when testing on Mac OSX, which doesn't support that option.